scripts/NuGetUtilities.psm1

```diff
@@ -0,0 +1,155 @@
+<#
+.SYNOPSIS
+    NuGet utility functions.
+
+.DESCRIPTION
+    The NuGetUtilities module exports functions for performing NuGet actions.
+#>
+
+Set-StrictMode -Version Latest
+$ErrorActionPreference = "Stop"
+$InformationPreference = "Continue"
+
+Import-Module $PSScriptRoot\ScriptUtilities.psm1
+Import-Module $PSScriptRoot\Projects.psm1
+
+$NugetExePath = "$RepoRoot\.nuget\NuGet.exe"
+$NuGetPackageRoot = "$SourceRoot\packages"
+$PackageSource = "https://nuget.org"
+$PackageOutputDirectoryRoot = Join-Path $BinRoot NuGet
+
+function Get-PackageVersion([switch]$previous) {
```

Comment:
>Get-PackageVersion [](start = 9, length = 18)

I extracted this code (which was previously inlined in `New-NuGetPackages`) into a function, because now it's also used in `Hide-NuGetPackages`. I needed to add the `-Previous` switch to pass along to `Get-VersionConstants`.